components/sensor/grove_gas_mc_v2.rst

```diff
@@ -0,0 +1,106 @@
+Grove Multichannel Gas Sensor V2
+================================================
+
+.. seo::
+    :description: Instructions for setting up Grove Multichannel Gas Sensor V2 that
+      can measure Nitrogen Dioxide, Carbon Monoxide, Ethanol and Volatile Organic
+      Compounds.
+    :image: grove-gas-mc-v2.png
+    :keywords: Grove, gm102b, gm302b, gm502b, gm702b
+
+The ``grove_gas_mc_v2`` sensor platform allows you to use your `Grove Multichannel Gas
+Sensor V2 <https://wiki.seeedstudio.com/Grove-Multichannel-Gas-Sensor-V2>`__ 
+with ESPHome. It exposes 4 different gas sensors for qualitatively measuring
+Nitrogen Dioxide (NO2), Carbon Monoxide (CO), Ethanol (C2H5OH), and Volatile Organic
+Compounds (VOCs).
+
+.. note::
+
+    The Grove Multichannel Gas Sensor V2 is a qualitative, not quantitative, sensor.
+    This means values reported back are raw ADC values. Values are **not** in a common unit
+    of measurement, such as PPM (parts per million). If you have known baseline readings
+    for any of the gases, :ref:`sensor-filters` could be used to calibrate the raw readings.
+    
+.. figure:: /images/grove-gas-mc-v2.png
+    :align: center
+    :width: 50.0%
+
+    Grove Multichannel Gas Sensor V2
+
+The communication with this sensor is done via :ref:`I²C Bus <i2c>`, so you need to have
+an ``i2c:`` section in your config for this integration to work.
+
+.. code-block:: yaml
+
+    i2c:
+      sda: D2
+      scl: D1
```

Comment:
No board specific pins here please. Use `GPIO` numbers or remove the i2c section as it is already stated above as a requirement.